src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: $backgroundcolor-page;
+  padding-left: $shift-sides-header;
+  padding-right: $shift-sides-header;
+  @extend %border-header;
+  &__icon {
+    height: $side-icon;
+    width: $side-icon ;
+    padding-top: $shift-top-icon;
+  }
+
+  &__nav {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+    align-items: flex-start;
+    padding: 0;
+    margin: 0;
+    background: $backgroundcolor-page;
+    @extend %border-header;
```

Comment:
Move this line, because you already have the same code in line 8. And it makes vertical shadow before list(on screen)
![Screenshot 2022-07-20 at 10 40 24](https://user-images.githubusercontent.com/17098736/179925207-70294a06-d771-468d-9e04-bb4a43b69c9d.png)

Reply:
I fixed